<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="robot_port" default="/dev/ttyACM2"/>  
  <arg name="robot_rate" default="57600"/> 
  <arg name="lidar_port" default="/dev/ttyUSB0"/>
  <arg name="tf_prefix_arg" default=""/>

  <node name="robot_node" pkg="robot_node" type="robot_node.py" output="screen">
    <param name="arduino_port" type="string" value="$(arg robot_port)"/>
    <param name="arduino_rate" type="int"    value="$(arg robot_rate)"/>
    <param name="tf_prefix" type="string"    value="$(arg tf_prefix_arg)"/>
  </node>



</launch>
